src/main/java/soolutions/hometaskapp/account/Account.java

```diff
@@ -2,9 +2,12 @@
 
 import soolutions.hometaskapp.common.Amount;
 
+import java.util.List;
+
 public interface Account {
     boolean open();
-    void deposit(Amount amount);
+    void apply(Transaction transaction);
 
+    List<Transaction> transactions();
```

Comment:
@SilasReinagel why `List`? Is there an ordering implicit here? Why not just `Collection`?

Reply:
Unit test won't compile without it. The transaction are currently in chronological order.